### PR TITLE
remove the "inject" function

### PR DESCRIPTION
**Pull request by eklitzke**
also remove global namespace pollution

### CODE REVIEW

File: ext/gh.js [hunk 1]
```diff
@@ -91,14 +91,15 @@ function gh_news() {
                         var $name = $('<a href="' + k + '">' + k + '</a>');
 
                         $title.append($name);
-                        var $event_count = '<span class="girdle_event_count"></span>';
+                        var $event_count = ('<span class="girdle_event_count">'
+                                            '</span>');
```
Comment (dforsyth) [inline ext/gh.js:95]: this seems to break the extension

File: ext/gh.js [hunk 2]
```diff
@@ -40,16 +40,16 @@ function gh_news() {
                     }
                     repo = $(key).text();
                 } else if (alert_type == 'alert issues_opened' ||
-                    alert_type == 'alert issues_closed' ||
-                    alert_type == 'alert issues_reopened') {
+                           alert_type == 'alert issues_closed' ||
+                           alert_type == 'alert issues_reopened') {
                     repo = $($(title_elems).get(2)).text();
                 } else if (alert_type == 'alert push' ||
```
Comment (paulmillr) [inline ext/gh.js:46]:
``` javascript
var addAlert = function(type) {return 'alert ' + type;};
var issueAlerts = ['issues_opened', 'issues_closed', 'issues_reopened'].map(addAlert);
var otherAlerts = ['push', 'commit_comment', 'download', 'delete', 'gollum', 'fork', 'watch_started'].map(addAlert);

...
else if (issueAlerts.indexOf(alertType) >= 0)
...
else if (otherAlerts.indexOf(alertType) >= 0)
```

**Comment by paulmillr** [comment]
:-1:, it's not only chrome users use this. I'm on safari, it doesn't support chrome exts.

**Comment by eklitzke** [comment]
I think I can redo this in a way that's hygienic for FF and Safari (I can only test on FF though), I plan to send another pull request.

**Pull request closed without merging**